index.js

```diff
@@ -99,10 +108,17 @@ class SimpleKeyring extends EventEmitter {
 
   /* PRIVATE METHODS */
 
-  _getWalletForAccount (account) {
+  _getWalletForAccount (account, opts) {
     const address = sigUtil.normalize(account)
     let wallet = this.wallets.find(w => ethUtil.bufferToHex(w.getAddress()) === address)
     if (!wallet) throw new Error('Simple Keyring - Unable to find matching address.')
+
+    if (opts.withAppKeyOrigin) {
```

Comment:
To keep `opts` optional, and avoid a major (breaking) version, I might keep an `if (!opts) opts = {}` higher up in this block.

Reply:
Done in bccf1d3